.ci/monolithic-linux.sh

```diff
@@ -63,9 +63,13 @@ pip install -q -r "${MONOREPO_ROOT}"/.ci/requirements.txt
 export LLVM_SYMBOLIZER_PATH=`which llvm-symbolizer`
 [[ ! -f "${LLVM_SYMBOLIZER_PATH}" ]] && echo "llvm-symbolizer not found!"
 
+# Set up all runtimes either way. libcxx is a dependency of LLDB.
+# If it ends up being unused, not much harm.
```

Comment:
I would change "not much harm" to explicitly enumerate that we do not end up configuring or building it if we we do not explicitly run those check/build targets.